newsApi.test.js

```diff
@@ -1,7 +1,16 @@
-// const NewsApi = require('./newsApi');
+const NewsApi = require('./newsApi');
+require("jest-fetch-mock").enableMocks();
 
-// describe("NewsApi class", () => {
-//   it("uses fetch to get article titles", () => {
+describe("NewsApi class", () => {
+  it("uses fetch to get article titles", () => {
+    const api = new NewsApi();
 
-//   });
-// });
\ No newline at end of file
+    fetch.mockResponseOnce(JSON.stringify({
+      response: { results: [ { webTitle : "Student decides mocking fetch is terrible" }]}
```

Comment:
Good job matching the structure of the response data to the actual data you would receive from the API.